indico/util/user.py

```diff
@@ -153,3 +155,25 @@ def wrapper(*args, **kwargs):
         return fn(*args, **kwargs)
 
     return wrapper
+
+
+def create_pending_user(user_data):
+    from indico.modules.users import User
+    from indico.modules.users.models.users import UserTitle
+
+    new_user = User()
+    new_user.email = user_data.get('email').lower()
+    new_user.first_name = user_data.get('firstName')
```

Comment:
Can't change it easily cause it can cause a lot of backward-compatibility related problems.